gulpfile.js

```diff
@@ -109,13 +109,13 @@ gulp.task('watch', () => {
 
   gulp.watch([
     'test/sandbox.html',
-    'dist/sweetalert2.all.min.js'
+    'dist/sweetalert2.js'
```

Comment:
you will need to watch `dist/sweetalert2.css` here as well

Reply:
done.